user/models.py

```diff
@@ -1,5 +1,6 @@
 from django.contrib.auth.models import AbstractUser
+from django.db import models
 
 
 class User(AbstractUser):
-    pass
+    is_staff = models.BooleanField()
```

Comment:
The `is_staff` field is already included in the `AbstractUser` model. Redefining it here is redundant and could cause confusion. Consider removing this line.